WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListView.swift

```diff
@@ -1,21 +1,33 @@
 import SwiftUI
 
-struct CustomFieldsDetailsView: View {
+struct CustomFieldsListView: View {
     @Environment(\.presentationMode) var presentationMode
 
     let isEditable: Bool
-    let customFields: [CustomFieldsViewModel]
+    let customFields: [CustomFieldViewModel]
 
     var body: some View {
         NavigationView {
             GeometryReader { geometry in
                 ScrollView {
                     VStack(alignment: .leading) {
                         ForEach(customFields) { customField in
-                            CustomFieldRow(isEditable: isEditable,
-                                           title: customField.title,
-                                           content: customField.content,
-                                           contentURL: customField.contentURL)
+                            if isEditable {
```

Comment:
If not editable, it would be nice to show an alert, explaining why

Reply:
`isEditable` here is simply a flag that contains the feature flag check while we're still developing. Once we're done, this will be removed as any field will be tappable (I see that it's confusing and probably should be `isTappable` instead of `isEditable`).

We do have a special type (JSON value) that can be tapped but can't be edited (refer to `isReadOnlyValue` param in `CustomFieldEditorView`). I agree we should let users know why this is not editable, whether with alert or a sort of label above the text box.